### ISSUE TITLE
Logging: Rename event_data members of skua_move event

### ISSUE THREAD

**Issue by LswaN58** [fielddaylab/penguinsVR#4]
For most of the events involving x/y/z coordinates, we used posX, posY, and posZ.
`skua_move` has both a "from" and "to" set of coordinates, so we need to take that into account when setting the names of each coordinate in the `event_data`. However, as written, we completely broke with the convention, using `from_position_x`, `to_position_x`, etc.
To keep better consistency between events, let's rename those to the form of `from_posX`, `to_posX`, etc

In other words, keep `position` shortened to `pos`, *don't* add an underscore between `pos(ition)` and `x`/`y`/`z`, and keep the coordinate letter upper-case (`X` instead of `x`).

**Comment by rossTnick** [comment]
4c7b677fdf1facdaf87022dcb57e4a57aa72ed33
